### PR TITLE
jainshubham1506.json

### PR DESCRIPTION
**Pull request by jainshubham1506**

### CODE REVIEW

File: jainshubham1506.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+username:jainshubham1506
```
Comment (crichID) [inline jainshubham1506.json]: @jainshubham1506 you will need to change this to `githubHandle:` instead of `username:`

Comment (crichID) [inline jainshubham1506.json]: @jainshubham1506 you will need to add a space after each `:`.

File: jainshubham1506.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+latitude:12.916517
+longitude:79.132499
```
Comment (crichID) [inline jainshubham1506.json]: @jainshubham1506 please add `---` as the last line of your file.

**Comment by brianamarie** [comment]
@jainshubham1506  I made some changes so the tests will pass. Once everything turns green, I'll go ahead and merge this in since it's been a few days and we want to keep the repository clean. 😄 ✨

**Pull request merged**